README.md

```diff
@@ -1,4 +1,6 @@
 lesson02
 ========
 
+![](https://www.loacker.com/uploads/pics/header_bildgalerie_welt_der_waffeln_classic_napolitaner_03.jpg?is_ilan=true&devour=yes)
```

Comment:
Where is the copyright def for this image?

Reply:
Inside, it is embedded - please check

Reply:
Can you please add an option to modify the image via the UI?
E.g. rotate, run a fourier transform etc, would be really helpful!

Reply:
Sure, please provide instructions on how to do so - I will gladly contribute further.

Reply:
Create a ui with an input
Then send the user request via sockets over UDP to the server
In the server create a worker which will parse the request and run the logic and will return the answer.
I'll be happy to have a technical design review with you once you will have a proper design.

Reply:
Its just a logo!

Reply:
I want it to be feature complete

Reply:
@ilantc can you please make the skeleton for me to fill the stuff in? I am looking forward to collaborating on this!

Reply:
@benjamingr feel free to create one :)
https://github.com/ilantc/lesson02/issues/13